environments/Mojaloop-Local.postman_environment.json

```diff
@@ -771,6 +771,11 @@
 			"key": "HOST_BULK_API_ADAPTER",
 			"value": "http://localhost:3003",
 			"enabled": true
+		},
+		{
+			"key": "HOST_TRANSACTION_REQUESTS_SERVICE",
+			"value": "http://transaction-request-service.local",
```

Comment:
transaction-requests-service.local I think ('s' after request)

Reply:
But in the helm ingress, it isn't for some reason.

https://github.com/mojaloop/helm/blob/master/transaction-requests-service/values.yaml#L88

Reply:
Yeah you're right, thats the value used in helm ingress..